packages/at_secondary_server/lib/src/verb/handler/pkam_verb_handler.dart

```diff
@@ -116,10 +172,9 @@ class PkamVerbHandler extends AbstractVerbHandler {
     }
     logger.finer('hashingAlgoEnum: $hashingAlgoEnum');
     final verificationInput = AtSigningVerificationInput(
-        utf8.encode('$sessionID$atSign:$storedSecret') as Uint8List,
+        utf8.encode('$sessionId$atSign:$storedSecret') as Uint8List,
         inputSignature,
         publicKey);
-    //
     verificationInput.signingAlgoType = signingAlgoEnum;
     verificationInput.hashingAlgoType = hashingAlgoEnum;
     verificationInput.signingMode = AtSigningMode.pkam;
```

Comment:
Before doing the signature verification, please handle conditions where any of signingAlgoEnum, inputSignature or hashingAlgoEnum are null. You can then add `!` where signingAlgoEnum, hashingAlgoEnum and inputSignature are referenced

Reply:
changed as part of [05f42a0](https://github.com/atsign-foundation/at_server/pull/1520/commits/05f42a080aa3b989ce7d6bddb7dbc3489b2825d5). The new methods ensure that the signingAlgoEnum and hashingAlgoEnum have default values, hence not nullable